app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -225,6 +230,16 @@ def create_vm(args = {})
       vm
     end
 
+    def preallocate_disks(args)
+      change_allocation_volumes = args[:volumes_attributes].values.select{ |x| x[:preallocate] == '1' }
```

Comment:
If I am creating a VM without disks, `args[:volumes_attributes]` is empty (until I something overlooked) and this would raise `NoMethodError: undefined method `values' for nil:NilClass`, am I right?

Reply:
yes, I'll fix it